armi/reactor/converters/uniformMesh.py

```diff
@@ -105,7 +138,20 @@ def _createNewAssembly(sourceAssembly):
         return a
 
     @staticmethod
-    def applyUniformMesh(sourceAssem, newMesh):
+    def makeAssemWithUniformMesh(sourceAssem, newMesh):
+        """
+        Build new assembly based on a source assembly but apply the uniform mesh.
+
+        The new assemblies must have appropriately mapped number densities as
+        input for a neutronics solve. They must also have other relevant
+        neutronics parameters for follow-on neutronics steps (e.g. a fixed
+        source gamma transport step that needs appropriate fission rates).
+        Thus, this maps many parameters from the ARMI mesh to the uniform mesh.
```

Comment:
As below this seems pretty neurtronics-y in a context where it shouldnt have to be. If this converter really is intended for neutronics-type stuff, perhaps we should move into that package.

Reply:
moved to a subclass at least for now. if this were  a plugin I'd namespace package it into `plugins.reactor.converters` but it'd feel a little weird to buy this down in physics.neutronics to me at the moment. Maybe you're right though